x-pack/plugins/observability/public/components/threshold/types.ts

```diff
@@ -45,10 +45,10 @@ export interface AlertContextMeta {
 
 export type MetricExpression = Omit<
   MetricExpressionParams,
-  'metric' | 'timeSize' | 'timeUnit' | 'metrics' | 'equation' | 'customMetrics'
+  'metric' | 'timeSize' | 'timeUnit' | 'metrics' | 'equation' | 'metrics'
 > & {
   metric?: NonCountMetricExpressionParams['metric'];
-  customMetrics?: CustomMetricExpressionParams['customMetrics'];
+  metrics?: CustomMetricExpressionParams['metrics'];
```

Comment:
I find naming to be confusing overall. For example, `CustomMetricExpressionParams` vs `MetricExpressionParams`. We should consider renaming all types derived from metric threshold rule to match with new rule name.

Reply:
Thanks for sharing the ticket. Looks like it will be covered as part of that ticket. 👍